plugins/remind.go

```diff
@@ -187,5 +245,5 @@ func (p *reminderPlugin) RemindCommand(b *seabird.Bot, m *irc.Message) {
 
 	b.MentionReply(m, "Event stored")
 
-	go p.dispatch(b, r)
+	p.updateChan <- struct{}{}
```

Comment:
`struct{}` is usually used because it's about as clear as possible. It's sending a notification with no data.